cmd/k8s-nameserver/main_test.go

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !plan9
+
+package main
+
+import (
+	"context"
+	"log"
+	"net/netip"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
I don't think this is documented explicitly (sorry), but I believe we're avoiding `testify` and using `frankban/quicktest` instead.